tensorflow/python/eager/context.py

```diff
@@ -1430,10 +1437,16 @@ def _initialize_physical_devices(self, reinitialize=False):
       self._physical_device_to_index = {
           p: i for i, p in enumerate(self._physical_devices)
       }
+      pluggable_devs = pywrap_tfe.TF_ListPluggablePhysicalDevices()
```

Comment:
Please add a comment that we maintain a separate list just so we can check which device is a PluggableDevice in `set_memory_growth`.